lib/shelley/src/Cardano/Wallet/Shelley/Pools.hs

```diff
@@ -494,7 +498,9 @@ monitorStakePools tr gp nl db@DBLayer{..} = do
     mkLatestGarbageCollectionEpochRef = newIORef minBound
 
     initCursor :: IO [BlockHeader]
-    initCursor = atomically $ readPoolProductionCursor (max 100 k)
```

Comment:
I think that the `readPoolProductionCursor` is made obsolete by this PR. So I'd be good to clean it up as well ?

Reply:
It is used in Jormungandr and I haven't touched that code yet.